strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/main/java/org/carlspring/strongbox/authentication/api/ldap/LdapAuthenticationConfigurationManager.java

```diff
@@ -146,7 +150,7 @@ public LdapConfiguration map(Map<String, Object> source)
 
         result.setUrl((String) source.get(URL));
         result.setManagerDn((String) source.get(MANAGER_DN));
-        result.setManagerPassword((String) source.get(MANAGER_PASSWORD));
+        result.setManagerPassword(String.valueOf(source.get(MANAGER_PASSWORD)));
```

Comment:
@anki2189 why this was changed?

Reply:
password can be 1234, it was failing in that case.

Reply:
ok, thanks